Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4055,7 +4055,7 @@ def merge_lists(original_list, updated_list, key):
     return merged_list
 
 
-def set_integration_context(context, sync=True, version=-1):
+def set_integration_context(context, version, sync=True):
```

Comment:
Why do we need this order change? We can still leave version with a default of -1 just treat it properly in the code.